### PR TITLE
fix build Makefile for macOS

### PR DESCRIPTION
**Pull request by erizocosmico**

### CODE REVIEW

**Comment by erizocosmico** [comment]
Included another fix, now it should be fine on macOS

**Comment by abeaumont** [comment]
@erizocosmico CI is failing

**Comment by erizocosmico** [comment]
@abeaumont bindata not being generated. It's just running `make`, right?

**Comment by juanjux** [comment]
@erizocosmico AFAIK just running `make` does the call to `go-bindata`.

File: etc/skeleton/driver/main.go.tpl [hunk 1]
```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"gopkg.in/bblfsh/sdk.v1/protocol/driver"
 
-	"github.com/bblfsh/{{.Manifest.Language}}-driver/driver/normalizer"
+	"gopkg.in/bblfsh/sdk.v1/etc/skeleton/driver/normalizer" //REPLACE:"github.com/bblfsh/{{.Manifest.Language}}-driver/driver/normalizer"
```
Comment (abeaumont) [inline etc/skeleton/driver/main.go.tpl]: This change shouldn't be here, since it refers to driver's module, not sdk's module.

**Comment by erizocosmico** [comment]
done @abeaumont

**Pull request merged**